### ISSUE TITLE
Windows 10 Pro Install Failure

### ISSUE THREAD

**Issue by GQM** [Storj/driveshare-gui#162]
I have been succesfully running DriveShare 0.4.2 for the end of December. Today I tried updating to DriveShare 0.5.0 and have been unsuccessful.

Upon installation things seem to install properly and installation GUI shows 100% complete. When I click close on the window it tries to "Execute: C:\Program Files (x86)\DriveShare\DriveShare.exe" which then causes the program to not respond. Even after a reboot, I try to run the program and it doesn't give me any sign its running and no error codes. What I do notice is that until you restart the computer you can't uninstall the program completely as it seems the .exe file is hanging in the background.

I've also tried starting from scratch by completely removing all DriveShare files from the AppData folder.

Find below my settings.json file as requested
```
{
  "tabs": [
    {
      "address": "1Jqa6BUx6adu7BSTtsUUGN6dFppZkL4hJQ",
      "storage": {
        "path": "\\\\NASF022F4\\Download\\Storj\\Drive0",
        "size": "7",
        "unit": "TB"
      },
      "id": "a93bea0f7f803cc2fdde06359f809097d326f2e8"
    },
    {
      "address": "1Jqa6BUx6adu7BSTtsUUGN6dFppZkL4hJQ",
      "storage": {
        "path": "\\\\NASF022F4\\Download\\Storj\\Drive1",
        "size": "7",
        "unit": "TB"
      },
      "id": "fee62c4cd2b7d64ae47d3fe1ffe3c11100abc8f8"
    }
  ]
}
```

**Comment by littleskunk** [comment]
Can't find a problem with your config. It works for me. Networkdrive is working. User without admin privileges can start it as well.

@gordonwritescode any ideas?

**Comment by gordonwritescode** [comment]
@GQM can you try to run DriveShare from the command line and paste any error messages you get here? 

I believe on windows, you can just open cmd.exe and execute `C:\Program Files (x86)\DriveShare\DriveShare.exe`.

**Comment by GQM** [comment]
Ok... so I have managed to run the GUI successfully by running it from source. What I noticed after is that when I went to install the GUI using the installer, this time the DriveShare window opened after clicking close on the install window. Unfortunately the window was blank. Then after closing it and trying to reopen it using the cmd line I just have a floating cursor with no error codes.

Going to try to restart the computer now and will report if anything had changed.

**Comment by GQM** [comment]
I can confirm that running the GUI from source is working and actively farming on the network using 0.5.0

Address: 1Jqa6BUx6adu7BSTtsUUGN6dFppZkL4hJQ

Also running the program still isn't working from the start menu, cmd line or program files.

I might try and build the source code into my own release later and see if the problem still exists.

**Comment by GQM** [comment]
Found the issue... my AntiVirus program has been blocking it as the program was from an unknown source. For some reason I wasn't being notified about this.

It was when I built it from source that I got a warning about the .exe file and was forced to turn off my protection to test. After this I can confirm the issue was related to my AntiVirus program. I have submitted a false positive claim and left my contact information for more details if they need it.
